docs/testing.md

```diff
@@ -47,15 +49,17 @@ Sauce Labs can be used to run tests remotely in the cloud.
 
 7. Run the tests with `gulp test:acceptance`.
 
-    > Note: If you want to temporarily disable testing on Sauce Labs,
-    run the command as: `gulp test:acceptance --sauce=false`.
+    !!! Note: 
```

Comment:
The `!!! Note` creates a callout when formatted by mkdocs, like this:

![image](https://cloud.githubusercontent.com/assets/10562538/17601125/943e8860-5fd4-11e6-992c-ec9a1c0558c1.png)